starboard/nplb/time_zone_get_current_test.cc

```diff
@@ -13,14 +13,58 @@
 // limitations under the License.
 
 #include "starboard/extension/time_zone.h"
+#include "starboard/nplb/time_constants.h"
 #include "starboard/system.h"
 #include "starboard/time_zone.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
+#include <array>
+#include <gmock/gmock.h>
+
 namespace starboard {
 namespace nplb {
 namespace {
 
+struct TimeZoneWithExpectValue {
+  TimeZoneWithExpectValue(std::string timeZoneName_,
+                          SbTimeZone expectedStandardValue_,
+                          SbTimeZone expectedDaylightValue_)
+      : timeZoneName{timeZoneName_},
+        expectedStandardValue{expectedStandardValue_},
+        expectedDaylightValue{expectedDaylightValue_} {}
+
+  std::string timeZoneName;
+
+  SbTimeZone expectedStandardValue;
+  SbTimeZone expectedDaylightValue;
+};
+
+class SbTimeZoneGetCurrentSetTimeZoneTest
```

Comment:
Drive-by comment: instead of adding this test logic here,  
could you add a new file with a more specific name, and  
add it to the BUILD.gn only if `is_win`? You can still  
leave an #ifndef(_WIN32) #error inside for good measure.